ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/PDEUIMessages.java

```diff
@@ -1092,7 +1092,11 @@ public class PDEUIMessages extends NLS {
 	public static String TracingTab_AttributeLabel_TracingOptions;
 	public static String TracingTab_AttributeLabel_TracingChecked;
 	public static String TracingTab_AttributeLabel_TracingNone;
+	public static String TracingTab_TracingOptionsManager_Progress;
```

Comment:
This has no value defined and is never called.
```suggestion
```

Reply:
Deleted